examples/quickstart/presence-sensor.js

```diff
@@ -13,6 +13,7 @@
  *
  * SPDX-License-Identifier: EPL-2.0 OR W3C-20150513
  ********************************************************************************/
+Object.defineProperty(exports, "__esModule", { value: true });
```

Comment:
This should not be there (according to the tests that I did in the past), I'll double-check locally to understand why it generates this line now.